config/daqsystemtest/moduleconfs.data.xml

```diff
@@ -345,7 +346,16 @@
  <attr name="generate_timesync" type="bool" val="1"/>
  <rel name="request_handler" class="RequestHandler" id="def-data-request-handler"/>
  <rel name="latency_buffer" class="LatencyBuffer" id="def-latency-buf"/>
- <rel name="data_processor" class="RawDataProcessor" id="def-wib-processor"/>
+ <rel name="data_processor" class="TPCRawDataProcessor" id="def-wib-processor"/>
+</obj>
+
+<obj class="DataHandlerConf" id="def-pds-tp-handler">
```

Comment:
why is there a different data handler conf for PDS TPs vs. TPC TPs? Shouldn't the PDS TPs still use the `def-tp-handler`?